priv/site/_includes/css/components/MobileNav.scss

```diff
@@ -93,3 +92,14 @@
     display: block;
   }
 }
+
+.MobileNav-alternate {
+  color: var(--c-white);
+  border-bottom: 1px solid white;
+  
+  svg {
+    path {
+      fill: var(--c-white);
+    }
+  }
+}
```

Comment:
Missing new line at end of file, you should take a look at your editor preferences, most likely you have the option to auto add the new line off.